src/stream/src/from_proto/dynamic_filter.rs

```diff
@@ -56,18 +56,40 @@ impl ExecutorBuilder for DynamicFilterExecutorBuilder {
             );
         }
 
-        // TODO: use consistent operation for dynamic filter <https://github.com/risingwavelabs/risingwave/issues/3893>
-        let state_table_l = StateTable::from_table_catalog_inconsistent_op(
+        let state_table_r =
+            StateTable::from_table_catalog(node.get_right_table()?, store.clone(), None).await;
```

Comment:
I believe this issue should be resolved? https://github.com/risingwavelabs/risingwave/issues/3893#issuecomment-1250497665

So just using `from_table_catalog` here, instead of `from_table_catalog_inconsistent_op`.

Reply:
cc @stdrc

Reply:
I think so.